api/v1/group_routes_test.go

```diff
@@ -64,15 +65,37 @@ func getControlSurface(tb testing.TB, testState *lib.TestRunState) *ControlSurfa
 func TestGetGroups(t *testing.T) {
 	t.Parallel()
 
+	cs := getControlSurface(t, getTestRunState(t, lib.Options{}, &minirunner.MiniRunner{}))
+	_ = cs.RunState.GroupSummary.Start()
```

Comment:
```suggestion
require.NoError(t, cs.RunState.GroupSummary.Start())
```

Even if it cannot fail, it's easier to follow, as you don't need to check the shape of what's being ignored, and why it will never fail.